core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.arraystostring;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JavaArraysToStringUnitTest {
```

Comment:
indent is 4 spaces

Reply:
As I'm using IntelliJ, I had some trouble with the XML not being applied; I used the Eclipse Formatter Plugin, let me know if it's okay.